src/Paket.Core/TemplateFile.fs

```diff
@@ -6,6 +6,80 @@ open System.IO
 open System.Text.RegularExpressions
 open Paket.Rop
 open Paket.Domain
+
+module private TemplateParser =
+    type private ParserState =
+        {
+            Remaining : string list
+            Map : Map<string, string>
+            Line : int
+        }
+
+    let private single = Regex("^(\S+)\s*$", RegexOptions.Compiled)
```

Comment:
I'm surprised that it actually does work like this, but you're right, it's likely cleaner to properly escape the backslashes. Anyway, you got it covered. ;)